<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Pradeepkulal-PC" failures="2" tests="11" name="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" time="712.576" errors="0" timestamp="2024-08-05T16:49:03 IST" skipped="2">
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="successfullyChangingPassword" time="77.059"/>
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="changePasswordWithMismatchedPasswordAndConfirmPassword" time="78.070"/>
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="successfullyChangingPasswordAfterFifteenMinutes" time="170.796">
    <failure message="expected [Password changed.] but found [Please wait 27 seconds before making another change password request.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Password changed.] but found [Please wait 27 seconds before making another change password request.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at org.intuitiveapps.Kycee.Tests.ChangePasswordTest.successfullyChangingPasswordAfterFifteenMinutes(ChangePasswordTest.java:79)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- successfullyChangingPasswordAfterFifteenMinutes -->
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="changePasswordUsingLastThreePassword" time="17.576">
    <skipped/>
  </testcase> <!-- changePasswordUsingLastThreePassword -->
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="LoginWithOldPaswordTest" time="18.129"/>
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="ChangingPasswordWithinFiveMinutes" time="71.806"/>
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="changePasswordWithEmptyConfirmNewPasswordfield" time="76.789"/>
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="changingPasswordWithInvalidOldPassword" time="70.318"/>
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="changePasswordWithEmptyOldPasswordfield" time="68.579">
    <failure message="expected [Please enter the valid new password.] but found [Please enter the valid old password.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [Please enter the valid new password.] but found [Please enter the valid old password.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at org.intuitiveapps.Kycee.Tests.ChangePasswordTest.changePasswordWithEmptyOldPasswordfield(ChangePasswordTest.java:43)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:664)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:228)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:961)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:201)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:819)
at org.testng.TestRunner.run(TestRunner.java:619)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1228)
at org.testng.TestNG.runSuites(TestNG.java:1134)
at org.testng.TestNG.run(TestNG.java:1101)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- changePasswordWithEmptyOldPasswordfield -->
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="changePasswordThreeTimesWithinTwentyFourHours" time="0.000">
    <skipped/>
  </testcase> <!-- changePasswordThreeTimesWithinTwentyFourHours -->
  <testcase classname="org.intuitiveapps.Kycee.Tests.ChangePasswordTest" name="changePasswordWithEmptyNewPasswordField" time="63.454"/>
</testsuite> <!-- org.intuitiveapps.Kycee.Tests.ChangePasswordTest -->
